### PR TITLE
[Feature] Add @everyminute to schedule_interval

### PR DESCRIPTION
**Pull request by nullhack**
**Description**

The only reason I can think to not include `@everyminute` on `schedule_interval`, is to keep consistency against `cron`. As we can see, all other relevant shortcuts are already [there](https://github.com/apache/airflow/blob/master/docs/dag-run.rst).
But they already are different (E.g. `@quarterly` and `@once` are not part of [cron](https://crontab.guru/) ).
I suggest including `@everyminute` as part of `presets` to make it complete.

**Testing code**
```python
from airflow import DAG
from airflow.utils.dates import days_ago
from datetime import timedelta
from airflow.operators.bash_operator import BashOperator

params = {
    'dag_id': 'test_minute',
    'schedule_interval': '@everyminute',
    'catchup': True,
    'max_active_runs': 2,
    'default_args': {
        'owner': 'airflow',
        'start_date': days_ago(1),
        'retries': 2,
        'retry_delay': timedelta(minutes=5)
    }
}

with DAG(**params) as dag:

    task_minute = BashOperator(
        task_id='task_minute',
        bash_command="""echo test """,
    )
    
    task_minute
```

**Edit**: The description has been changed from `@minutely` to `@everyminute` to comply with the suggestions on comments.

**Sample output**
image
image

---
Make sure to mark the boxes below before creating PR: [x]

- [X] Description above provides context of the change
- [X] Unit tests coverage for changes (not needed for documentation changes)
- [X] Target Github ISSUE in description if exists
- [X] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [X] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

### CODE REVIEW

**Comment by nullhack** [comment]
@mik-laj sorry, don't know any other maintainers yet to ask for review XD may I ask you to review (or tag someone to)?

It's a small change, but my colleagues liked the idea. I'm sending It back to the community.

File: airflow/utils/dates.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 from airflow.utils import timezone
 
 cron_presets = {
+    '@minutely': '* * * * *',
```
Comment (mik-laj) [inline airflow/utils/dates.py]: The word "minutely" does exist, 

https://english.stackexchange.com/questions/3091/weekly-daily-hourly-minutely

Comment (nullhack) [reply airflow/utils/dates.py]: If I understood correctly the word exists, but is archaic/has double meaning [[1]](https://www.thefreedictionary.com/minutely) [[2]](https://www.merriam-webster.com/dictionary/minutely) [[3]](https://www.dictionary.com/browse/minutely?s=t)
If this is the case, I don't think double meaning is a problem here and the word just fits nice with the rest (but English is not my first language, so I'll leave the debate to natives)

I can just close the PR, but personally I think some preset for minutes is missing. It's not about the size either, It is easier to visually find on the UI, so I would not mind having a bigger string that describes It.

Would other strings be considered ?
`@every_minute`, `@all_minutes`, `@minutes`

Comment (JeffryMAC) [reply airflow/utils/dates.py]: May I ask what kind of dag you are scheduling every minute? It sounds very overwhelming. airflow sometimes take more than a minute just to take the task from None to queue state.
Do you have something like this running in your production?

Comment (nullhack) [reply airflow/utils/dates.py]: In my particular case, some REST requests doing routine checks. Not part of any heavy process. We haven't reach the point of taking minutes to schedule/run tasks yet, only about 50 DAGs and maximum around 20 tasks per DAG, most of It running daily.

Comment (mik-laj) [reply airflow/utils/dates.py]: I'm not a native speaker either, so it's difficult for me to review this change. I just noticed that this word is not common. I will wait for the opinions of other people.

**Comment by ashb** [comment]
I'm -0.5 on this change. It can be achieved with `schedule_interval="* * * * *"` or `schedule_interval=timedelta(minutes=1)`.

File: airflow/utils/dates.py (hunk 1 above)
Comment (ashb) [reply airflow/utils/dates.py]: `@everyminute` if we do decide to keep it.

**Comment by nullhack** [comment]
I've changed the PR to `@everyminute` as suggested, If the members decide to keep the changes.

**Pull request closed without merging**